test/e2e/specs/editor/various/autocomplete-and-mentions.spec.js

```diff
@@ -426,13 +460,14 @@ test.describe( 'Autocomplete', () => {
 
 	test( 'should hide UI when selection changes (by mouse)', async ( {
 		page,
+		editor,
 	} ) => {
-		await page.click( 'role=button[name="Add default block"i]' );
+		await editor.canvas.click( 'role=button[name="Add default block"i]' );
 		await page.keyboard.type( '@fr' );
 		await expect(
 			page.locator( 'role=option', { hasText: 'Frodo Baggins' } )
 		).toBeVisible();
-		await page.click( '[data-type="core/paragraph"]' );
+		await editor.canvas.click( '[data-type="core/paragraph"]' );
 		await expect(
 			page.locator( 'role=option', { hasText: 'Frodo Baggins' } )
 		).not.toBeVisible();
```

Comment:
Is this a related failure?

Reply:
Tests are now run in the iframe, so yes.